ERC20_LIST.json

```diff
@@ -391,14 +437,29 @@
         "useInSwap": false,
         "wrappingMethod": "BorrowStaker"
       },
-      "0x7e880867363A7e321f5d260Cade2B0Bb2F717B02": {
```

Comment:
Why deleting this one from the token list?

Reply:
because it seems like a typo, this is not a token. It's convex base reward pool for the FRAXBP, so I don't see where it could be use. Correct me if I am wrong @Picodes?